apps/src/applab/dropletConfig.js

```diff
@@ -337,6 +337,15 @@ export var blocks = [
     params: ['250', '0', '75', '0.5'],
     type: 'value'
   },
+  {
+    func: 'open',
+    parent: api,
+    category: 'UI controls',
+    paletteParams: ['url'],
+    params: [
+      '"https://en.wikipedia.org/w/api.php?origin=*&action=parse&format=json&prop=text&page=computer&section=1&disablelimitreport=true"'
```

Comment:
I'm sure you can change this later, but just wanted to call out that this example might give the wrong idea -- If I understand correctly, this block will more often be used to take the user to a human-readable page like https://en.wikipedia.org/wiki/Computer